### ISSUE TITLE
xbox 0x87e00017 error

### ISSUE THREAD

**Issue by ddev01** [ChrisTitusTech/winutil#598]
**Describe the bug**
TLDR Can't install xbox games and get 0x87e00017 error code.

I used the 'Desktop recommended tweaks' and all the MISC tweaks besides Disable TPM, notifications and set time to UTC. I got a free xbox game pass subscription so I tried installing some games and keep getting the 0x87e00017 error when trying to install a game. 

I googled it online and noticed some people received this error after using o/o shutup. I used undo tweaks which didnt fix it and also downloaded the o/o tool and set it to factory/default settings but I'm still unable to download games. 

ofcourse I have tried other fixes unrelated to this tool like resetting xbox, gamebar, xbox live, troubleshooting them, manually enabling UAC, using windows store apps troubleshooter, resetting microsoft store, checking for updates on microsoft store etc.

**Comment by d0x360** [comment]
you need ALL xbox and gaming related items, if undoing it didnt help then install windows again and next time make a system image first
